src/components/Emphasizewidget.tsx

```diff
@@ -32,10 +32,10 @@ const Emphasizewidget = () => {
         SetQueryList(newList);
     }
 
-    function emphasizeQuery(_event: React.MouseEvent<HTMLButtonElement, MouseEvent>): void { 
-        updateEmphasis (queryList);
+    async function emphasizeQuery(_event: React.MouseEvent<HTMLButtonElement, MouseEvent>): Promise<void> { 
+        const queryErrors = await updateEmphasis (queryList);
```

Comment:
You are not using queryErrors to do anything.  How do you update the status of the query if you do not do anything with the errors?